modules/local/make_asv_count_tables/main.nf

```diff
@@ -0,0 +1,23 @@
+
+process MAKE_ASV_COUNT_TABLES {
+    tag "$meta.id"
+    label 'light'
+    conda "${projectDir}/conf/environment.yml"
+
+    input:
+    tuple val(meta), path(maps), path(asvtaxtable), path(reads), path(extracted_var_path)
+
+    output:
+    tuple val(meta), path("*asv_krona_counts.txt"), emit: asv_count_tables_out
+
+    """
+    if [[ ${meta.single_end} = true ]]; then
+        zcat $reads | sed -n "1~4p" > headers.txt
+        python /hps/software/users/rdf/metagenomics/service-team/users/chrisata/asv_gen/bin/make_asv_count_table.py -t $asvtaxtable -f $maps -a $extracted_var_path -hd ./headers.txt  -s ${meta.id}
```

Comment:
this python script is part of the toolkit, this hps path is temporary because I have pending changes to it that I don't want to release yet just in case I need to make some more changes to them in this code review, but good catch!